scripts/on_env_start.bat

```diff
@@ -56,6 +56,10 @@ if "%update_branch%"=="" (
 @copy sd-ui-files\scripts\check_modules.py scripts\ /Y
 @copy "sd-ui-files\scripts\Start Stable Diffusion UI.cmd" . /Y
 @copy "sd-ui-files\scripts\Developer Console.cmd" . /Y
+if exist modifiers\ {
```

Comment:
These .bat scripts will need to be tested by someone on Windows. I only have Linux to test these changes